modules/assim.sequential/R/Analysis_sda.R

```diff
@@ -227,8 +238,19 @@ GEF<-function(setting,Forecast,Observed, H, extraArg, nitr=50000, nburnin=10000,
     }
     
   }
-
-  dat.tobit2space <- runMCMC(Cmcmc_tobit2space, niter = nitr, nburnin=nburnin,  progressBar=TRUE)
+  
+  if(file.exists(file.path(outdir, paste0('dat.tobit2space',t,'.Rdata')))){
+    load(file.path(outdir, paste0('dat.tobit2space',t,'.Rdata')))
+  }else{
+    dat.tobit2space <- runMCMC(Cmcmc_tobit2space, niter = nitr, nburnin=nburnin,  progressBar=TRUE)
+    save(dat.tobit2space, file = file.path(outdir, paste0('dat.tobit2space',t,'.Rdata')))
+  }
+  
+  pdf(file.path(outdir,paste0('assessParams',t,'.pdf')))
+  try(assessParams(dat = dat.tobit2space[1000:4000,], Xt = X))
```

Comment:
In the multi-site I ask users to define the length of the chains, That'd be great if you could not fix these numbers.